### ISSUE TITLE
[Feature request] add range selector buttons

### ISSUE THREAD

**Issue by michapr** [aukedejong/lovelace-windrose-card#41]
Would it be possible to add range selectors like for "plotly-graph-card"?:

![image](https://github.com/aukedejong/lovelace-windrose-card/assets/11854556/7444d9f0-a5ae-4bac-845b-4168a3cfe65a)

see 
https://github.com/dbuezas/lovelace-plotly-graph-card

So it would be possible to check the wind direction for different interval - last 30minutes, last hour etc.

THANKS!

**Comment by Aaroneisele55** [comment]
I would also like this to be added, but if that's not possible, I'd like to have the period value templatable so I could at least jerry-rig myself a range selector with an input_select...

**Comment by gedger** [comment]
Me too, even adding an option to show what the current range is  would be very helpful.

**Comment by aukedejong** [comment]
Feature added in version 1.16.0

**Comment by il77781** [comment]
@aukedejong, thank you!

**Comment by aukedejong** [comment]
Ah yes. I did not yet try it with that many button, good idea.
Will be in next version.

**Comment by aukedejong** [comment]
Fixed in 1.16.1

**Comment by il77781** [comment]
@aukedejong, yes, now everything is fine with "multi-row" buttons! Thank you!

**Comment by paulbputter** [comment]
How do I use this feature?

**Comment by aukedejong** [comment]
Example in this post:

https://community.home-assistant.io/t/lovelace-windrose-card/523319/260?u=aukedejong
